client/blocks/app-promo/test/index.jsx

```diff
@@ -1,18 +1,13 @@
+/** @jest-environment jsdom */
```

Comment:
I can confirm that is slower when you pick `jsdom` as a default test env:

> Test Suites: 7 failed, 928 passed, 935 total
> Tests:       28 failed, 10 skipped, 8707 passed, 8745 total
> Snapshots:   0 total
> Time:        65.348s
> Ran all test suites.
> 
> real    1m6.598s
> user    7m5.317s
> sys     0m22.786s

On the other hand, we would have to use this notation only in 7 files. We can revisit it later.